### DISCUSSION TITLE
Launch Deezer playlist from HA stopped working

### DISCUSSION THREAD

**Discussion by Toukite** [alandtse/alexa_media_player#1986] [General]
**IMPORTANT: Please search the issues, including closed issues, and the [FAQ](https://github.com/custom-components/alexa_media_player/wiki/FAQ) before opening a new issue. The template is mandatory; failure to use it will result in issue closure.**

**Describe the bug**

Launch Deezer playlist from HA stopped working

**To Reproduce**

1. Create Deezer Playlist named "Ma Playlist Test" with one track in it
2. Create script as below and launch it

**Expected behavior**

Track should play on alexa device

**Screenshots**

**System details**

- Home-assistant (version): latest
- alexa_media (version from `const.py` or HA startup): latest
- alexapy (version from `pip show alexapy` or HA startup): latest
- Amazon [2FA is enabled](https://github.com/custom-components/alexa_media_player/wiki/Configuration#enable-two-factor-authentication-for-your-amazon-account) (y/n). : yes mfa enabled

**Logs**
Please provide [logs](https://github.com/custom-components/alexa_media_player/wiki/FAQ#how-do-i-enable-debug-logging-for-the-component).
No logs found

**Additional context**

Hello,

French user here with Deezer as music service. 
I'm wondering about following script that stopped working a few months ago:

service: media_player.play_media
data:
..entity_id: media_player.echodot5
..media_content_id: Ma playlist Test
..media_content_type: DEEZER
enabled: true

It used to play the only track present in this playlist. 
I remember having trouble setting that up (naming of the playlist needed to be exactly like that or it did not find it, and it could only have 1 track in it, etc, but finally the above script worked)

A few months ago it stopped working, now Alexa says "I can't do that at the moment" and doesn't play the sound anymore.

Not a frequent use case and I guess not so much people prefer Deezer over Spotify but ... still asking :) 
Anybody can help ?

Thank you !

**Comment by alandtse** [comment]
Converting to discussion for workarounds. Nothing we can do if Amazon deprecates a service.
